### DISCUSSION TITLE
Is it possible Docker without WSL?

### DISCUSSION THREAD

**Discussion by ironcg20** [docker-library/mongo#670] [General]
I want to use Docker directly on Windows systems, without relying on a Linux-based OS like Ubuntu. Can Docker be used in this way? I don't have a deep understanding of the technicalities involved, so I'm asking both developers and users for their input.

**Comment by yosifkit** [comment]
Yes and no. You can run [Windows-based Docker containers](https://learn.microsoft.com/en-us/virtualization/windowscontainers/about/) without WSL (or a Linux VM). You cannot run Linux-based containers without WSL (or a Linux VM). Most container images are Linux based, though a few have Windows variants (that might work similarly).

**Reply by ironcg20** [reply]
Thank you for your response. Some individuals have reported system corruption when utilizing Docker on Windows. 
Could you please explain in detail? This is very important and serious for me.

**Reply by whalelines** [reply]
It is not possible for us to respond to nonspecific concerns, and this is not the appropriate forum for such requests. This repository is specifically concerned with the MongoDB official image, https://hub.docker.com/_/mongo . If you have specific questions, you can get help at the following links:

1. MongoDB: https://www.mongodb.com/services/support
2. Docker: https://www.docker.com/support/
